tests/clients/web/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
 # All of our core javascript will be checked with a single test
-add_javascript_style_test(core src)
```

Comment:
Yes, I think you're right.  That's one of the problems I've found with eslint; it doesn't error when you give it a non-existent path.  I'll fix it up later tonight.

As far as linting the test files, I think that would be most appropriate for a separate PR.